.github/workflows/gh-pages.yml

```
@@ -42,19 +37,45 @@ jobs:
         run: |
           cp -R ~jekyll-dist/_includes/* _includes/
           cp -R ~jekyll-dist/_layouts/* _layouts/
-          sed -i '/^dist\/$/s/^/#/' .gitignore
 
-      - name: Commit all changes
-        run: |
-          git config --global user.name "${{ github.repository_owner}}"
-          git config --global user.email "${{ github.actor_id }}+${{ github.actor }}@users.noreply.github.com"
-          git add .
-          git commit -m "${{ github.run_id }}: Deploy to gh-pages" --allow-empty
+      # Necessary in order to build Jekyll
+      - name: Setup Ruby
+        # https://github.com/ruby/setup-ruby/releases/tag/v1.207.0
+        uses: ruby/setup-ruby@4a9ddd6f338a97768b8006bf671dfbad383215f4
+        with:
+          ruby-version: '3.1' # Not needed with a .ruby-version file
```

Looks like #2381 bumped it up already
```suggestion
          ruby-version: '3.3'
```